client/app/telecom/telephony/line/phone/configuration/translations/Messages_fr_FR.xml

```diff
@@ -196,4 +196,10 @@
    <translation id="telephony_line_phone_configuration_config_ldap_server_address">LdapServerAddress</translation>
    <translation id="telephony_line_phone_configuration_config_ldap_server_port">LdapServerPort</translation>
    <translation id="telephony_line_phone_configuration_config_ldap_username">LdapUsername</translation>
+
+<!-- SOFT KEY LABEL -->
+   <translation id="telephony_line_phone_configuration_config_soft_key_label_1">SoftKeyLabel1</translation>
```

Comment:
Does it make sense to generate like `SoftKeyLabel{{ version }}` ?

Reply:
Yes it would be, but we receive the complete label from the API.